packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -1127,6 +1121,98 @@ enum BoxShape {
   circle
 }
 
+/// An immutable set of radii for each corner of a rectangle. Only used with
+/// [BoxShape.rectangle].
+class BorderRadius {
+  /// Creates a border radius with all radii are [radius].
+  const BorderRadius.all(Radius radius) : this.only(
+    topLeft: radius,
+    topRight: radius,
+    bottomRight: radius,
+    bottomLeft: radius
+  );
+
+  /// Creates a vertically symmetrical border radius where the top and bottom
```

Comment:
We usually use the word "symmetric", not "symmetrical" (though both are correct)